package.json

```diff
@@ -45,11 +46,11 @@
     "find-file-in-dirs": "^1.0.0",
     "initial-commit-date": "^1.0.0",
     "is-there": "^4.0.0",
-    "jade": "^1.11.0",
     "markdox": "^0.1.10",
     "ncp": "^2.0.0",
     "one-by-one": "^3.1.1",
     "parent-search": "^1.3.1",
+    "pug": "^2.0.0-alpha6",
```

Comment:
Ahh, that's a good point! :+1: Let's just nuke this dependency. :fire: I just realized we only use ejs. I remember in the past I played with `jade`, but like you said, it is not used anymore.